rest-server/data/ur.json

```diff
@@ -3,21 +3,22 @@
   "engineFactory": "com.actionml.engines.ur.UREngine",
   "sparkConf": {
     "master": "yarn",
+    "appName": "harness-spark-test-app",
+    "mongo.database": "harness_meta_store",
```

Comment:
This is not needed is it, in any case it should not be defined here but discovered from harness, maybe by a call to the administrator? Why is it needed? Ideally it is private to the administrator.

Reply:
@pferrel I've changed appName to `engineId`
Collection (and database it belongs to) is needed by spark job to create RDD from it

Reply:
ok. I have struggled with what to name things like DBs, etc. We only have the engineId that is guaranteed to be unique but we can append things to this to make it more descriptive if needed. Like engineId+"training" or smth.